pkg/ibmvpc/connectivity_result.go

```diff
@@ -21,3 +27,88 @@ type ConnectivityResult struct {
 	deniedConns map[*ipblock.IPBlock]*connection.Set // denied target and its allowed connections, by deny rules.
 	denyRules   map[*ipblock.IPBlock][]int           // indexes of deny rules relevant to this connectivity
 }
+
+func equalKeys[T any](first, second map[*ipblock.IPBlock]T) bool {
+	if len(first) != len(second) {
+		return false
+	}
+	keys1 := make([]string, len(first))
+	i := 0
+	for ipBlock := range first {
+		keys1[i] = ipBlock.String()
+		i += 1
+	}
+	i = 0
+	sort.Strings(keys1)
```

Comment:
this part is also duplicated.. can be moved to a separate function that returns a list of sorted keys from such a map.